code/modules/modular_computers/file_system/programs/pai/radio.dm

```diff
@@ -44,7 +44,7 @@
 			"listening" = (host.radio.channels[ch_name] & host.radio.FREQ_LISTENING)
 		)
 		pai_channels[++pai_channels.len] = channel_info
-	VUEUI_SET_CHECK(data["channels"], pai_channels, ., data)
+	data["channels"] = pai_channels
```

Comment:
If you're going to do this, you need to always return data. It won't return data currently if *only* pai_channels changes.
Recommend removing the VUEUI_SET_CHECKS and just doing `return data` at the end.

Technically we are already always returning data already due to how header data works (which is bad but we've got it *everywhere* now...)